### ISSUE TITLE
[Bug]: cannot paste bibliography

### ISSUE THREAD

**Issue by kzssc** [chrisgrieser/alfred-bibtex-citation-picker#56]
### Bug Description

the bibliography is not pasted when triggered. 

### To Reproduce

choose any entry ->  hold option+cmd+enter -> does not paste.  / pastepin receives no input from workflow. 

### Debugging Log

```Text
[10:00:29.779 pm] Supercharged Citation Picker[Script Filter] Passing output 'susskind2020WorldWorkTechnology' to Run Script
[10:00:29.780 pm] ERROR: Supercharged Citation Picker[Run Script] Task '/Users/vpchris/SynologyDrive/Dropbox/Alfred.alfredpreferences/workflows/user.workflow.781461E7-FDDD-4C5D-AF98-0EDB380512DD/scripts/paste-full-bibliography.sh' failed with reason 'launch path not accessible'
[10:00:29.781 pm] Supercharged Citation Picker[Run Script] Processing complete
[10:00:29.792 pm] Supercharged Citation Picker[Run Script] Passing output '' to Copy to Clipboard
```

### Sample BibTeX Entries

```BibTeX
@book{1992SelfEmploymentPrograms,
  title = {Self {{Employment Programs}} for {{Unemployed Workers}}},
  year = {1992},
  publisher = {{U.S. Dept. of Labor, Employment and Training Administration, Unemployment Insurance Service}},
  address = {Washington, D.C.},
  langid = {english},
  annotation = {OCLC: 30034279},
  file = {/Users/vpchris/Zotero/storage/J8X8KID2/Wandner et al 1992 Self Employment Programs for Unemployed Workers.pdf}
}

@book{2002RightSelfEmployment,
  title = {Right to {{Self Employment Bill}}},
  year = {2002},
  publisher = {House of Parliament, Great Britain},
  isbn = {978-0-215-70097-1},
  langid = {english},
  keywords = {5},
  annotation = {OCLC: 1116137684},
  file = {/Users/vpchris/Zotero/storage/7JL54C5T/Great Britain et al 2002 Right to Self Employment Bill.pdf}
```

### Buffer Files

[last_version.txt](https://github.com/user-attachments/files/17347045/last_version.txt)
[buffer.json](https://github.com/user-attachments/files/17347046/buffer.json)

### Workflow Configuration

<img width="1315" alt="image" src="https://github.com/user-attachments/assets/69ce90d5-5b0e-4434-9163-16fff472268e">

### Checklist

- [X] I have [updated to the latest version](https://github.com/chrisgrieser/alfred-bibtex-citation-picker/releases/latest) of this workflow.
- [X] I have tried manually reloading the buffer (Alfred keyword `scp`, select `Force Buffer Reload`).

**Comment by chrisgrieser** [comment]
I added some improved logging. Please download the latest release and try again:
https://github.com/chrisgrieser/alfred-bibtex-citation-picker/releases/download/11.15.6/alfred-bibtex-citation-picker.alfredworkflow

Also, please run `which -a pandoc` in the terminal and tell me what the output is?

**Comment by kzssc** [comment]
issue fixed in 11.15.6 (not sure why). 

re terminal:

/opt/homebrew/bin/pandoc
/usr/local/bin/pandoc

**Comment by chrisgrieser** [comment]
okay, in that case it is likely that you deleted a file of the workflow on your end. `launch path not accessible` normally only occurs when a file Alfred is trying to run is missing. Probably you accidentally deleted something when you where checking out the workflow. Please do not change anything in the workflow's folder, since that usually breaks Alfred workflows and is also incredibly time-consuming to figure out.
